### PR TITLE
UPDATE XPM FILES:

### PR DESCRIPTION
**Pull request by Elena-Lou**
The xpm files for the textures are updated to fit the required 64*64 format.

### CODE REVIEW

File: srcs/raycasting/dda_rays.c [hunk 1]
```diff
@@ -99,6 +99,7 @@ void	ft_vrt_intersections(t_cub_data *data, t_dda *ray)
 
 void	ft_cast_ray(t_cub_data *data, t_dda *ray)
 {
+	printf("RAY ID[%d]\n", ray->id);
```
Comment (AdrianWeaver) [inline srcs/raycasting/dda_rays.c:102]: You might want to remove these printf before merge ?

Also you have a WIP commit not squashed before merge.

**Review by AdrianWeaver — CHANGES_REQUESTED** [review]
Please consider squashing the WIP commit and also removing the printf debugs.

File: include/cub3d.h [hunk 2]
```diff
@@ -133,6 +133,7 @@ typedef struct s_dda
 	double	hzt_dist;
 	double	vrt_dist;
 	double	strip_height;
+	int		wall_impact;
```
Comment (AdrianWeaver) [inline include/cub3d.h:136]: So this is the x coords relative to the wall, right?

**Review by AdrianWeaver — APPROVED** [review]

File: include/cub3d.h (hunk 2 above)
Comment (Elena-Lou) [reply include/cub3d.h:136]: yes it is !

**Pull request merged**